node/src/actors/chain_manager/mod.rs

```diff
@@ -827,8 +837,10 @@ impl ChainManager {
             // of the ARS, even if the superblock hash is different from our local superblock hash.
             // If the superblock index is different from the current one we cannot check ARS membership,
             // so we broadcast it if the index is within an acceptable range (not too old).
-            let should_broadcast = match act.chain_state.superblock_state.add_vote(&superblock_vote)
-            {
+            let should_broadcast = match act.chain_state.superblock_state.add_vote(
+                &superblock_vote,
+                act.current_epoch.unwrap() / superblock_period,
```

Comment:
`act.current_epoch` can be `None` if the current epoch is below epoch 0. I think here we can safely set the epoch to 0 in that case:

```suggestion
                    act.current_epoch.unwrap_or(0) / superblock_period,
```